src/components/Dashboard/Pay.tsx

```diff
@@ -106,6 +113,16 @@ export default function Pay() {
 
   return (
     <div>
+      <div style={{ marginBottom: '0.5rem' }}>
+        {tokenSymbol && tokenAddress && (
```

Comment:
nit pick—could we move line 117 outside of the parent div, so we aren't creating any new empty containers if there's no token?

Reply:
Lovely suggestion, thanks!

Reply:
https://github.com/jbx-protocol/juice-interface/pull/311/commits/32821cbcbc0f50a9eb0c8fe14740b46185b00df0